loopy/symbolic.py

```diff
@@ -169,9 +182,7 @@ def map_resolved_function(self, expr, *args, **kwargs):
     map_rule_argument = map_group_hw_index
 
     def map_fortran_division(self, expr, *args, **kwargs):
-        return type(expr)(
-                self.rec(expr.numerator, *args, **kwargs),
-                self.rec(expr.denominator, *args, **kwargs))
+        return self.map_quotient(expr, *args, **kwargs)
```

Comment:
Wouldn't this use map_quotient from child class too?
I think just `map_fortran_division = map_quotient` should fix that.

Reply:
Thanks for pointing this out! Held up the merge to wait on a fix for this.

Reply:
Yep, will do that.